pyHS100/pyHS100.py

```diff
@@ -57,6 +63,39 @@ def __init__(self, ip_address, protocol=None):
             protocol = TPLinkSmartHomeProtocol()
         self.protocol = protocol
 
+        self.schedule = schedule.Schedule(self, self._query_helper)
+        self.countdown = countdown.Countdown(self, self._query_helper)
+        self.antitheft = antitheft.AntiTheft(self, self._query_helper)
+
+    def new_schedule(self):
+        """
+        Used by Schedule instance when it gets deleted.
+
+        :return: None
+        :rtype: None
+        """
+
+        self.schedule = schedule.Schedule(self, self._query_helper)
+
+    def new_countdown(self):
+        """
+        Used by Countdown instance when it gets deleted.
+
+        :return: None
+        :rtype: None
+        """
+
+        self.countdown = countdown.Countdown(self, self._query_helper)
+
+    def new_antitheft(self):
+        """
+        Used by AntiTheft instance when it gets deleted.
+        :return: None
+        :rtype: None
+        """
+
+        self.antitheft = antitheft.AntiTheft(self, self._query_helper)
+
```

Comment:
up to you. i can also create a specific method for deleting all rules as well without the need to loop.